examples/web_ui/playwright/playwright_async/playwright_test_async.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
+from playwright.async_api import async_playwright
+from guara.transaction import Application, AbstractTransaction
```

Comment:
There is an issue here. You need to import the Application `from guara.asynchronous.transaction import Application`. check this [example](https://github.com/douglasdcm/guara/blob/main/examples/web_ui/caqui_async/asynchronouos/test_async.py#L14), please